### PR TITLE
[REDO AFTER GH BUG] Add support for quantized models via QNN

### PR DESCRIPTION
**Pull request by masahi**
This is part of https://github.com/apache/incubator-tvm/issues/5015. I simply reverted the revert done in #5013 .

@anijain2305 @jwfromm @jjohnson-arm Since the original PR, I finished preparing the repo for imagenet evaluation. We can use the 1k subset (data auto downloaded), or full validation set. The script and updated numbers are available [here](https://github.com/Edgecortix-Inc/pytorch_quantization/tree/master/tvm_qnn_evaluation).

### CODE REVIEW

**Comment by masahi** [comment]
Thanks @anijain2305 @jwfromm
